README.md

```diff
@@ -21,6 +21,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.2/install.html#
 | `bbb_letsencrypt_email` | E-mail for use with letsencrypt | | _(required when letsencrypt is enabled)_ |
 | `bbb_nginx_privacy` | only log errors not access | `yes` |
 | `bbb_nginx_listen_https` | nginx: use https | `yes` | This is useful for a reverse proxy configuration where the BBB server is behind a load balancing server like haproxy that does SSL termination |
+| `bbb_nginx_root` | Defaut nginx www path of BigBlueButton | `/var/www/bigbluebutton-default` |
```

Comment:
Comment missing for README `| Your Comment |` it's not needed but nice to have.
For example a hint that no trailing `/` is needed and even forbidden.

Reply:
Thank you for reviewing the PR.
Fixed in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/107/commits/c5d0db9bbe001d864902bca9374595da8cebb53b